### PR TITLE
Do not return private data in public profile

### PR DESCRIPTION
**Pull request by eodolphi**
- Create new private, authenticated profile endpoint
 - Remove private fields from public endpoint
 - Make public endpoint read only
 - Add tests

### CODE REVIEW

File: bluebottle/bb_accounts/serializers.py [hunk 1]
```diff
@@ -103,12 +102,23 @@ class UserProfileSerializer(serializers.ModelSerializer):
...
-                  'campaign_notifications', 'project_count', 'donation_count',
-                  'fundraiser_count', 'task_count', 'time_spent')
+        fields = ('id', 'url', 'full_name', 'short_name', 'picture',
+                  'primary_language', 'about_me', 'location',
+                  'project_count', 'donation_count', 'date_joined',
+                  'fundraiser_count', 'task_count', 'time_spent',
+                  'website', 'twitter', 'facebook', 'skypename', )
+
+
+class UserPrivateProfileSerializer(UserProfileSerializer):
```
Comment (gannetson) [inline bluebottle/bb_accounts/serializers.py]: Maybe use the pattern ManageProfileSerializer like me do with ManageTask, ManageProject etc. ?

Comment (eodolphi) [inline bluebottle/bb_accounts/serializers.py]: Do you mean naming wise? Or do the ManageSerializers do other special things?

Comment (gannetson) [inline bluebottle/bb_accounts/serializers.py]: Just the name yeah

Comment (eodolphi) [inline bluebottle/bb_accounts/serializers.py]: Will do.

**Comment by gannetson** [comment]
Should point to release.
Will fire a new PR fot that.

**Pull request closed without merging**